src/routes/home.tsx

```diff
@@ -83,8 +83,15 @@ export default function HomeRoute() {
                   <div className="justify-start items-center gap-2 flex">
                     <Dropdown
                       inline
-                      label={`Show ${sort}`}
-                      size="sm"
+                      label={
+                        <p>
+                          Sort by{' '}
+                          <span className="text-sm font-normal leading-tight pl-1">
+                            {sort}
+                          </span>
+                        </p>
+                      }
+                      size="xs"
```

Comment:
I think  `<p>Sort by{' '}` should exist outside <Dropdown> to better match the intent of #95.

Same for the Show Dropdown.